src/_guides/language/language-tour.md

```diff
@@ -3547,6 +3554,65 @@ enum Color { red, green, blue }
   to help prevent copy-paste errors.
 {{site.alert.end}}
 
+#### Declaring enhanced enums
+
+Dart also allows enum declarations to declare classes
+with fields, methods, and const constructors
+which are limited to a fixed number of known constant instances.
+
+To declare an enhanced enum,
+follow a syntax similar to normal [classes](#classes),
+but with a few extra requirements:
+
+* Instance variables must be `final`, 
+  including those added by [mixins](#mixins).
+* All constructors must be
+  [constant generative constructors](#constant-constructors).
+* No other class can be extended, [`Enum`] is the superclass.
+* There cannot be overrides for `index`, `hashCode`, the equality operator `==`.
+* A member named `values` cannot be declared in an enum,
+  as it would conflict with the automatically generated static `values` getter.
+* All instances of the enum must be declared
+  in the beginning of the declaration
+  (of which there must be at least one).
```

Comment:
"of which" reads like it refers to "declaration", not "instances".
Maybe

 (and there must be at least one instance declared).